### PR TITLE
Render Point Share fixes

### PR DESCRIPTION
**Pull request by kwnie**
## Description

This is to get FE running. There was an extra dependency array that is not needed, and 'authState' needs to be changed to 'user' as part of the Auth0 changes.

### CODE REVIEW

**Review by da-vazquez — APPROVED** [review]
Great job Kelsey! Thank you for resolving this render issue so quickly

**Comment by da-vazquez** [comment]
One thing to note, on line 46 the setVirtualPlayerPoints function is declared but never used again so this may be something to address in the future

File: src/components/pages/PointShare/RenderPointShare.js [hunk 1]
```diff
@@ -109,31 +108,23 @@ const PointShare = props => {
     if (teamPoints) {
       submitPoints(user, teamPoints);
     }
-  }, [teamPoints, user]);
-  
+
```
Comment (Antashma) [inline src/components/pages/PointShare/RenderPointShare.js:111]: Great that for catching useEffect and the authState error!

**Review by Antashma — APPROVED** [review]

**Review by kdneldor — APPROVED** [review]
Looks great! Thank you for this!

**Pull request merged**